.github/workflows/tests.yml

```diff
@@ -0,0 +1,16 @@
+name: Tests
+on: pull_request
+jobs:
+  Tests:
+    name: Check that all tests runs successfully
+    runs-on: ubuntu-20.04
+    container: fedora:34
+    steps:
+      - name: Install Pipenv and Git
+        run: dnf install -y pipenv git
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Setup environment
+        run: pipenv sync --dev
+      - name: Run Tests
+        run: pipenv run pytest -v
```

Comment:
Why is that in this pr? we will add that in this PR:  #36

Reply:
when the pr will be merged this will be rebased and removed